### PR TITLE
Add a Contribs Project for Prospective EarlGrey API's.

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

File: Demo/EarlGreyContribs/EarlGreyContribsSwiftTests/EarlGreyContribsTestRig-Bridging-Header.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import EarlGrey;
+
+#import "MainViewController.h"
```
Comment (bootstraponline) [inline Demo/EarlGreyContribs/EarlGreyContribsSwiftTests/EarlGreyContribsTestRig-Bridging-Header.h]: no newline at end of file

Comment (tirodkar) [inline Demo/EarlGreyContribs/EarlGreyContribsSwiftTests/EarlGreyContribsTestRig-Bridging-Header.h]: I'll add in newlines everywhere. Thanks!

File: Demo/EarlGreyContribs/EarlGreyContribsTests/EarlGreyContribsTests.pch [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#ifdef __OBJC__
+@import EarlGrey;
+@import Foundation;
+@import UIKit;
+@import XCTest;
+#endif
```
Comment (bootstraponline) [inline Demo/EarlGreyContribs/EarlGreyContribsTests/EarlGreyContribsTests.pch]: missing newline

**Comment by bootstraponline** [comment]
Looks awesome! This uses EarlGrey from master, right?

File: Demo/EarlGreyContribs/EarlGreyContribsTests/EarlGreyContribsTests.pch (hunk 2 above)
Comment (tirodkar) [inline Demo/EarlGreyContribs/EarlGreyContribsTests/EarlGreyContribsTests.pch]: Done. Also added a newline for EarlGrey.swift in the gem.

**Comment by tirodkar** [comment]
> Looks awesome! This uses EarlGrey from master, right?

Yes, you need to use the setup-script in order to get it running.

**Pull request merged**